api/package.json

```diff
@@ -17,7 +17,6 @@
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "i": "^0.3.7",
-    "n": "^8.0.2",
```

Comment:
I don't think we need this here?  [This looks like](https://www.npmjs.com/package/n) a package that should be installed globally (on your local machine rather than in the project - i.e. `npm install -g n`)?

Reply:
Resolved.